iphone/Classes/TiAppiOSProxy.m

```diff
@@ -126,6 +126,9 @@ - (void)_listenerAdded:(NSString *)type count:(int)count
                                                  name:kTiApplicationLaunchedFromURL
                                                object:nil];
   }
+  if (count == 1 && [type isEqual:@"userinteraction"]) {
+      [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleUserInteraction:) name:@"kTiUserInteraction" object:nil];
+    }
 }
```

Comment:
Please move over to the Ti.App namespace. Also, lint all sources using `clang-format -style=file -i Classes/<file>.m` from the `/iphone` folder.

Reply:
Done!